backend/server/src/main/java/com/issuetracker/dto/ios/IosLabelDto.java

```diff
@@ -0,0 +1,69 @@
+package com.issuetracker.dto.ios;
+
+import com.issuetracker.domain.Label;
+
+public class IosLabelDto {
+
+    private Long id;
+
+    private String title;
+
+    private String description;
+
+    private String backgroundColorHexa;
+
+    private String textColorHexa;
```

Comment:
```suggestion
public class IosLabelDto {

    private final Long id;

    private final String title;

    private final String description;

    private final String backgroundColorHexa;

    private final String textColorHexa;
```

dto 는 기본적으로 불변 객체라고 생각하는 편이 좋습니다.
정보의 변경이 필요하다면, 새로운 dto 객체를 만들어야지 기존의 dto 객체를 수정해서는 안됩니다.
html 을 수정할 때는 html 을 새로 렌더링한다는 느낌으로 접근해야합니다.

Reply:
오늘 오프라인에서 설명 너무 잘해주셔서 이해할 수 있었습니다.
DTO는 값을 담기 위한 VO 중 하나 이지만, 
다른 점은  다른 계층으로 데이터(정보)를 보내기 위해서 캡슐화하기 때문에 VO와 달리, 잭슨또는 restTemplate과 반드시 호환 돼야한다! 입니다!